### DISCUSSION TITLE
The style of the PDF printed table does not match the size of the previous page's style and the next page's style

### DISCUSSION THREAD

**Discussion by 858882847** [eclipse-birt/birt#1410] [Q&A]
![BM2M{JAKOKP5 N{OS5J{RDQ](https://github.com/eclipse-birt/birt/assets/117696273/241cc0d6-67f0-4eef-bf41-216bed85f9b7)

**Comment by 858882847** [comment]
[new_report_1.zip](https://github.com/eclipse-birt/birt/files/12509280/new_report_1.zip)
How to solve the problem of inconsistent content in birt print styles

**Comment by hvbtup** [comment]
Your report contains errors:

A BIRT exception occurred. See next exception for more information.
Error evaluating Javascript expression. Script engine error: ReferenceError: "paramList" is not defined. (/report/data-sets/script-data-set[@id="85"]/method[@name="open"]#3)
 Script source: /report/data-sets/script-data-set[@id="85"]/method[@name="open"], line: 0, text:
__bm_OPEN(). (Element ID:1)

And I can't reproduce the issue.

**Comment by 858882847** [comment]
> 您的报告包含错误：
> 
> 发生了 BIRT 异常。有关详细信息，请参阅下一个异常。计算 Javascript 表达式时出错。脚本引擎错误： 引用错误： 未定义“参数列表”。（/report/data-sets/script-data-set[=“85”]/method[=“open”]#3）脚本源：/report/data-sets/script-data-set[=“85”]/method[=“open”]，行：0，文本：__bm_OPEN（）。（元素 ID：1）
> 
> 而且我无法重现该问题。
[new_report_1.zip](https://github.com/eclipse-birt/birt/files/12511275/new_report_1.zip)

I can print, it's normal for me to print vertically, but this happens when printing horizontally

**Comment by hvbtup** [comment]
I still can't reproduce the issue and I'm not sure I understand the issue. Can you attach the generated PDF file?

**Comment by 858882847** [comment]
> 我仍然无法重现该问题，并且不确定我是否理解该问题。您可以附加生成的PDF文件吗？

[tz.zip](https://github.com/eclipse-birt/birt/files/12511990/tz.zip)

**Comment by 858882847** [comment]
> I still can't reproduce the issue and I'm not sure I understand the issue. Can you attach the generated PDF file?

The upper and lower structures of horizontal printing are inconsistent, especially the columns on the last page are asymmetric with those on the previous page

**Comment by hvbtup** [comment]
Clarification: The pages are A4 landscape, 3 pages total. On the 3rd page, the table is enlarged in comparison to pages 1 and 2. It is not only wider, even the font size is seems increased.

**Comment by hvbtup** [comment]
Neither the table nor the columns have a width defined; the master pages uses A4 landscape, no margins defined.
The font and its size are not defined.
So the report uses default values for everything.
I think that the correct rendering is in fact page 3, because the default width for a table should be to use 100% of the available width.

I saw that the report was created with BIRT 4.8.0. Can you reproduce the issue with a 4.14 snapshot?

**Comment by 858882847** [comment]
> 表和列都没有定义宽度;母版页使用 A4 横向，未定义边距。未定义字体及其大小。因此，报表对所有内容都使用默认值。我认为正确的渲染实际上是第 3 页，因为表格的默认宽度应该是使用可用宽度的 100%。
> 
> 我看到该报告是使用 BIRT 4.8.0 创建的。您可以使用 4.14 快照重现该问题吗？

Due to JDK reasons, I can only temporarily use version 4.8, and other people are also using version 4.8 without any issues

**Comment by hvbtup** [comment]
> Due to JDK reasons, I can only temporarily use version 4.8, and other people are also using version 4.8 without any issues

But you can try to reproduce the issue with 4.14 and tell us the results.

**Comment by speckyspooky** [comment]
Which way you have used to create the PDFdocument, do you used the 4.8 all-in-one-designer?

I checked your report with BIRT 4.6, 4.8, 4.10, 4.12, 4.14 (latest built) and I wasn't able according to Henning to get your PDF version.
